_episodes/58.md

```diff
@@ -0,0 +1,173 @@
+---
+layout: episode
+title: "58: Understanding your users with Trino and Mitzu"
+date: 2024-04-04
+tags: trino
+youtube_id: "jKg17UXEVjg"
+wistia_id: "jp2x892ck8"
+sections:
+- title: "Introduction"
+  desc: "Episode 57: Understanding your users with Trino and Mitzu"
+  time: 0:00
+- title: "Releases"
+  desc: "Trino 442, 443, and 444"
+  time: 1:16
+- title: "Trino Gateway"
+  desc: "Version 7 and Starburst Trino Champion Peng Wei"
+  time: 5:23
+- title: "Starburst Trino Tamas Cservenak"
+  desc: "For all the Apache Maven help"
+  time: 6:09
+- title: "Introduction of István Mészáros"
+  desc: "Founder and CEO of Mitzu"
+  time: 8:12
+- title: "Mitzu introduction and logo"
+  desc: ""
+  time: 9:35
+- title: "High level overview "
+  desc: "Mitzu and warehouse-native product analytics"
+  time: 10:36
+- title: "What products and what source data?"
+  time: 12:04
+- title: "More about events, users, and sessions"
+  time: 13:30
+- title: "Mitzu use cases, personas, and difference to BI tools"
+  time: 15:30
+- title: "Common and supported event data structures"
+  time:  17:10
```

Comment:
Not sure if it matters but this has an extra bit of spacing.

Reply:
Fixed